internal/config/pipelineconfig.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+// PipelineConfig is the manually-maintained configuration for the pipeline.
+type PipelineConfig struct {
+	// The name of the image to use, where the convention is not
+	// appropriate. The tag is specified in PipelineState.
+	ImageName string `json:"image_name,omitempty"`
```

Comment:
Question, do we want/need yaml tags on this too to ensure casing? Or does it just work as expected already?

Reply:
We need eventually. But this change did not touch the PipelineConfig part of the code, (This appears in the CL because of file renaming).
I think it should be done together with the redefine work tracked in https://github.com/googleapis/librarian/issues/773

Reply:
ack